GeneralStateTests/stPreCompiledContracts/modexp_0_0_0_1000000.json

```diff
@@ -0,0 +1,159 @@
+{
+    "modexp_0_0_0_1000000" : {
+        "env" : {
+            "currentCoinbase" : "0x3535353535353535353535353535353535353535",
+            "currentDifficulty" : "0x1",
```

Comment:
this is a wrong header for blockchain test. 
I guess we could remove some of the fields in env section as for the blockchain test it should always be 

            "currentCoinbase" : "address",
            "currentDifficulty" : "0x020000",
            "currentGasLimit" : "anything < 2**63-1 but make sure the transaction does not hit",   
            "currentNumber" : "1",
            "currentTimestamp" : "1000",